pylatex/utils.py

```diff
@@ -249,6 +249,25 @@ def bold(s, *, escape=True):
     return NoEscape(r'\textbf{' + s + '}')
 
 
+def text_color(s, color, *, escape=True):
```

Comment:
Can you make this a ContainerCommand in basic as well?

If a special package is needed for the textcolor command can you add that to packages.